bucket/prismlauncher-qt5.json

```
@@ -1,10 +1,21 @@
 {
-    "version": "5.2",
-    "description": "An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability. Qt 5 build that still supports legacy Windows like 8.1 and 7.",
+    "version": "6.0",
+    "description": "An open sourced Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability. Qt 5 build that still supports legacy Windows like 8.1 and 7.",
     "homepage": "https://prismlauncher.org/",
     "license": "GPL-3.0-only",
-    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/5.2/PrismLauncher-Windows-Legacy-5.2.zip",
-    "hash": "74b1154c64b206f5a64bf965e9383a606461605ee92a609192aa0f67d8a6efd3",
+    "notes": "To add Prism Launcher file association options for .ZIPs and .MRPACKS, run this: \"$dir\\install-associations.reg\"",
+    "url": "https://github.com/PrismLauncher/PrismLauncher/releases/download/6.0/PrismLauncher-Windows-MSVC-Legacy-6.0.zip",
+    "hash": "68b8ea5211c3cb6041b5f4dffb4a95bae80a649ab04d7b58b0bb8648b6e7204f",
+    "post_install": [
+        "'install-associations', 'uninstall-associations' | ForEach-Object {",
+        "    if (Test-Path \"$bucketsdir\\games\\scripts\\prismlauncher-qt5\\$_.reg\") {",
+        "        $exePath = \"$dir\\prismlauncher.exe\".Replace('\\', '\\\\')",
+        "        $content = (Get-Content \"$bucketsdir\\extras\\scripts\\prismlauncher-qt5\\$_.reg\").Replace('$EXEPATH', $exePath)",
```

```suggestion
        "        $content = (Get-Content \"$bucketsdir\\games\\scripts\\prismlauncher-qt5\\$_.reg\").Replace('$EXEPATH', $exePath)",
```
I assume this is still copied from the windows-terminal I linked for the scripts? Needs to be `games`, not `extras` pretty sure. As suggested by 404networkError

---

whoops sorry